components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildValidationDataCommand.java

```diff
@@ -124,9 +124,9 @@ private void logException(RuntimeException e) {
 
     private void printFailedRequest(FailedRequestException e) {
         logger.error("Response status code: " + e.httpStatusCode());
-        if (e.getResponseBody() != null) {
+        if (e.getResponseBody().isPresent()) {
```

Comment:
Use `Optional.ifPresent` instead to avoid separate calls to `isPresent()` and `get()`.

Reply:
Fixed in 7a8ad80427b5d98488c58e495b2ea3e84602202d